Framework/include/Utility.h

```diff
@@ -23,6 +25,13 @@ namespace utility
     template<typename T> T sum2(T v) { return v*v; }
     template<typename T, typename... Args> T sum2(T v, Args... args) { return v*v + sum2(args...); }
     template<typename T, typename... Args> T addInQuad(T v, Args... args) { return sqrt(sum2(v, args...)); }
+    template<typename LV1, typename LV2>
+    double calcMT(const LV1& lepton, const LV2& met)
```

Comment:
Do we need to calculate MT somewhere? Also, I love the use of a template here.

Reply:
Special case reference in `Analyze1Lep` for calculating `mT` for use in the long ago W control region. Presumably reference to W control region could be removed.

Reply:
It's ok to leave it. I was just curious. Good template function practice.

Reply:
Turns out this method is a bit more useful now as I need to calculated the MT between a muon and MET for the Top tagger semileptonic selection and the `TLorentzVector` method simply does not work in this specific case.